### PR TITLE
[DAT-12759] Add logic to include TableSpace into Table during snapshot for DB2Z

### PR DESCRIPTION
**Pull request by andrulban**

### CODE REVIEW

File: liquibase-core/src/main/java/liquibase/structure/core/Table.java [hunk 1]
```diff
@@ -31,6 +31,15 @@ public List<ForeignKey> getOutgoingForeignKeys() {
         return getAttribute("outgoingForeignKeys", List.class);
     }
 
+    public String getTableSpace() {
```
Comment (achehrynets) [inline liquibase-core/src/main/java/liquibase/structure/core/Table.java]: We need to rename this method to `getTablespace`
To make it consistent with other classes

File: liquibase-core/src/main/java/liquibase/structure/core/Table.java [hunk 2]
```diff
@@ -31,6 +31,15 @@ public List<ForeignKey> getOutgoingForeignKeys() {
...
+        return getAttribute("tableSpace", String.class);
+    }
+
+    public Relation setTableSpace(String remarks) {
```
Comment (achehrynets) [inline liquibase-core/src/main/java/liquibase/structure/core/Table.java]: We need to rename this method to `setTablespace`
To make it consistent with other classes

File: liquibase-core/src/main/java/liquibase/structure/core/Table.java [hunk 3]
```diff
@@ -31,6 +31,15 @@ public List<ForeignKey> getOutgoingForeignKeys() {
         return getAttribute("outgoingForeignKeys", List.class);
     }
 
+    public String getTableSpace() {
+        return getAttribute("tableSpace", String.class);
```
Comment (achehrynets) [inline liquibase-core/src/main/java/liquibase/structure/core/Table.java]: Let's rename it from `tableSpace` to `tablespace`

File: liquibase-core/src/main/java/liquibase/structure/core/Table.java [hunk 4]
```diff
@@ -31,6 +31,15 @@ public List<ForeignKey> getOutgoingForeignKeys() {
...
+    }
+
+    public Relation setTableSpace(String remarks) {
+        setAttribute("tableSpace", remarks);
```
Comment (achehrynets) [inline liquibase-core/src/main/java/liquibase/structure/core/Table.java]: Let's rename it from `tableSpace` to `tablespace`

**Review by achehrynets — CHANGES_REQUESTED** [review]
@andrulban , left a few comments

File: liquibase-core/src/main/java/liquibase/structure/core/Table.java (hunk 1 above)
Comment (andrulban) [reply liquibase-core/src/main/java/liquibase/structure/core/Table.java]: Done, thanks!

File: liquibase-core/src/main/java/liquibase/structure/core/Table.java (hunk 2 above)
Comment (andrulban) [reply liquibase-core/src/main/java/liquibase/structure/core/Table.java]: Done, thanks!

File: liquibase-core/src/main/java/liquibase/structure/core/Table.java (hunk 3 above)
Comment (andrulban) [reply liquibase-core/src/main/java/liquibase/structure/core/Table.java]: Done, thanks!

File: liquibase-core/src/main/java/liquibase/structure/core/Table.java (hunk 4 above)
Comment (andrulban) [reply liquibase-core/src/main/java/liquibase/structure/core/Table.java]: Done, thanks!

**Review by achehrynets — APPROVED** [review]

**Pull request merged**